library/core/src/slice/mod.rs

```diff
@@ -3069,19 +3069,18 @@ impl<T> [T] {
         sort::unstable::sort(self, &mut |a, b| f(a).lt(&f(b)));
     }
 
-    /// Reorders the slice such that the element at `index` after the reordering is at its final
-    /// sorted position.
+    /// Reorders the slice such that the element at `index` is at a sort-order position. All
+    /// elements before `index` will then be `<=` this value, and all elements after will be `>=`.
```

Comment:
This (and the other occurrences) read a little awkwardly to me. Maybe "will be `<=` *to* this value"?

Reply:
Good catch!  Pushed a commit addressing this.

Reply:
Sorry, I think you misunderstood what I meant (though that's also a good catch). I was highlighting the missing "to":
```diff
- will be `<=` this value
+ will be `<=` to this value
```
Maybe it is just me, but the sentence flows a little oddly without it. I guess I am just reading the symbols in my head instead of "greater than or equal to". Similarly,
```diff
- all elements after will be `>=`
+ all elements after will be `>=` to it.
```